pwa/src/styling/design-tokens/gateway-ui-design-tokens.css

```diff
@@ -50,11 +53,23 @@
 
   /* Colors */
   --gateway-ui-color-white: #ffffff;
+  --gateway-ui-color-black: black;
+  --gateway-ui-color-darkgrey: darkgrey;
+  --gateway-ui-color-red: red;
+  --gateway-ui-color-dark-red: rgb(146, 2, 2);
+  --gateway-ui-color-green: green;
+  --gateway-ui-color-status-green: #69b090;
+  --gateway-ui-color-status-red: #dd3c49;
+  --gateway-ui-color-method-get: var(--gateway-ui-color-status-green);
+  --gateway-ui-color-method-post: #51b8f4;
+  --gateway-ui-color-method-put: #e9743a;
+  --gateway-ui-color-method-delete: var(--gateway-ui-color-status-red);
```

Comment:
We don't need this one, we can just call the `--gateway-ui-color-error` from the other comment.